### PR TITLE
Feat/life parameters

### PR DESCRIPTION
**Pull request by meczajkowski**

### CODE REVIEW

**Review by NerdbordMentor — CHANGES_REQUESTED** [review]
## Code Review Report

### Task goal
Tamagotchi life parameters should decrease in time

### Requirements
1. `energy` should decrease by 1 point per 2 seconds
2. `hunger` should decrease by 1 points per second
3. `fun` should decrease by 1 points per second
4. `health` should decrease by 1 points per second when `hunger` or `energy` is below or equal to 0
5. `energy` should decrease by additional 1 point per 2 seconds when fun is below or equal to 0

### Code changes review

#### app.js
The changes in this file are correct and meet the requirements. The game is now started with all the necessary elements.

#### index.html
The changes in this file are correct and meet the requirements. The HTML elements for displaying the Tamagotchi's parameters have been correctly identified with classes.

#### js/game.js
The changes in this file are correct and meet the requirements. The game now starts with all the necessary elements.

#### js/modules/tamagotchi.js
The changes in this file mostly meet the requirements, but there's one mistake:

- In the `mount` method, `this.displayHunger` is called three times instead of calling `this.displayEnergy` and `this.displayFun`. This should be corrected as follows:

```diff
- this.displayHunger(energyElement);
- this.displayHunger(funElement);
+ this.displayEnergy(energyElement);
+ this.displayFun(funElement);
```

Other than that, the code correctly implements the logic for decreasing the Tamagotchi's parameters according to the rules specified in the task requirements.

**Review by NerdbordMentor — APPROVED** [review]
## Code Review Report

### Task goal
Tamagotchi life parameters should decrease in time

### Requirements
1. `energy` should decrease by 1 point per 2 seconds
2. `hunger` should decrease by 1 points per second
3. `fun` should decrease by 1 points per second
4. `health` should decrease by 1 points per second when `hunger` or `energy` is below or equal to 0
5. `energy` should decrease by additional 1 point per 2 seconds when fun is below or equal to 0

### Review

The submitted code changes cover the task requirements.

1. The `energy` parameter is decreased by 1 point per 2 seconds as required. This is implemented in the `decreaseParameters` function in the `tamagotchi.js` file.

2. The `hunger` parameter is decreased by 1 point per second as required. This is implemented in the `decreaseParameters` function in the `tamagotchi.js` file.

3. The `fun` parameter is decreased by 1 point per second as required. This is implemented in the `decreaseParameters` function in the `tamagotchi.js` file.

4. The `health` parameter is decreased by 1 point per second when `hunger` or `energy` is below or equal to 0 as required. This is implemented in the `decreaseParameters` function in the `tamagotchi.js` file.

5. The `energy` parameter is decreased by an additional 1 point per 2 seconds when fun is below or equal to 0 as required. This is implemented in the `decreaseParameters` function in the `tamagotchi.js` file.

The code changes also include updates to the HTML and JavaScript files to display the updated parameters on the webpage, which was not explicitly required but is necessary for the functionality of the game.

The code changes do not introduce any apparent bugs or issues, and they adhere to good coding practices. The code is well-structured and easy to read, with clear comments explaining the functionality of each section.

In conclusion, the code changes meet all of the task requirements and are ready to be merged into the main codebase.
